src/lTerm_widget.mli

```diff
@@ -253,6 +256,31 @@ class ['a] radiobutton : 'a radiogroup -> string -> 'a -> object
 
 end
 
+(** {6 Scrollbars} *)
+
+(** methods required for a widget to supprt scrollbars *)
+class type scrollable = object
+  method full_size : LTerm_geom.size
+    (* full size of scrollable window *)
+
+  method offset : LTerm_geom.coord
```

Comment:
This API is quite specific for the case with an horizontal scrollbar and/or a vertical one. I'm wondering if an API similar to lablgtk wouldn't be better. i.e. instead of having widget satisfy the `scrollable` interface, you would do: `blah#set_vadjustment vscroll`. If the scrollbar new more informations such as the range, it could use that for drawing a smaller or longer bar for instance